tox_poetry_installer/installer.py

```diff
@@ -46,6 +47,13 @@ def install(
 
     installed: Set[PoetryPackage] = set()
 
+    def logged_install(dependency: PoetryPackage) -> None:
+        start = datetime.now()
+        logger.debug(f"Installing {dependency}")
+        pip.install(dependency)
+        end = datetime.now()
+        logger.debug(f"Finished installing {dependency} in {end - start}")
```

Comment:
Could the execution time be formatted so that it displays cleanly rather than the `datetime.timedelta` repr?

Reply:
It should be the `datetime.timedelta` str:
```python
>>> from datetime import datetime
>>> start = datetime.now()
>>> end = datetime.now()
>>> start
datetime.datetime(2021, 9, 10, 10, 23, 21, 534647)
>>> end
datetime.datetime(2021, 9, 10, 10, 24, 55, 58994)
>>> f"Finished installing foo in {end - start}"
'Finished installing foo in 0:01:33.524347'
```
Is that okay, or do you want something more explicit (n hours, m minutes, s seconds)?

Reply:
Nope, the built in string format will work fine. I think I just don't trust the datetime module so assumed it wouldn't be well formatted :sweat_smile: